sklearn/decomposition/_nmf.py

```diff
@@ -1081,7 +1081,7 @@ def non_negative_factorization(X, W=None, H=None, n_components=None, *,
 
 
 class NMF(TransformerMixin, BaseEstimator):
-    r"""Non-Negative Matrix Factorization (NMF)
+    """Non-Negative Matrix Factorization (NMF)
```

Comment:
Could you please revert the raw string directive? I think this is the reason of the failing of some of the checks. Thanks!

Reply:
This is definitly where the problem comes from but if I revert to r doc string another error will raise because the of backslashes "\" on lines 934 and 1154. The problem is that in this docstring the "\" is used both in the latexlike comments and as a return line sign. I didn't find a way to fix the problem, the only way would be to make an exception on the length of the line and keep lines 934/935 and 1154/1155 on the same row. What do you think could be done ?

Reply:
@climbst you are right.
As the latex symbol `\sum` is not meant to be rendered as latex (see the [correspondent documentation](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.NMF.html#sklearn.decomposition.NMF)), do you mind checking if rewriting the two lines
```
||A||_Fro^2 = \sum_{i,j} A_{ij}^2 (Frobenius norm)
||vec(A)||_1 = \sum_{i,j} abs(A_{ij}) (Elementwise L1 norm)
``` 
as
```
||A||_Fro^2 = sum(A_{ij}^2) (Frobenius norm)
||vec(A)||_1 = sum(abs(A_{ij})) (Elementwise L1 norm)
``` 
will fix the issue?
@glemaitre WDYT? Thanks.

Reply:
Since this already not human-readable in its simple format, we could use the `math` environment from RST such that sphinx will display properly on the HTML format. Then, if we need a raw string, we can have it.

Reply:
It's worth giving `math` RST formatting a try but I am not sure it will fix the orginal `\` issue.

I think one would have to remove the raw `r` marker and then double math `\`, no? It's a bit ugly but it should work.

Reply:
Escaping the backslashes withing math RST formatting seems to work. 
I had to to choose between framing the equations has it was done before or having latex formulas (couldn't combine the two). What do you think ?